.changeset/serious-drinks-check.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Tweaks to the v2 catalog ingestion.
```

Comment:
We don't have any changesets for the v2 catalog yet, prolly best to just put this in the PR body for now to track it? It's gonna look a bit weird to have this be the first v2 entry in the changelog of the catalog backend.